tests/unit/testLogger.js

```diff
@@ -15,65 +15,55 @@
  */
 
 'use strict';
-var test = require('tape');
+var assert = require('assert');
 var lodash = require('lodash');
 var createLogger = require('../../src/logger.js');
 
-test('Initializing the logger', function (t) {
-  var oldEnv = process.env.GCLOUD_ERRORS_LOGLEVEL;
-  delete process.env.GCLOUD_ERRORS_LOGLEVEL;
-  t.doesNotThrow(createLogger, createLogger(),
-    'Does not throw given undefined');
-  t.doesNotThrow(createLogger.bind(null, {}), createLogger(),
-    'Does not throw given an empty object');
-  t.doesNotThrow(
-    createLogger.bind(null, {logLevel: 3}),
-    createLogger({logLevel: 3}),
-    'Does not throw given a valid configuration object with a valid log level'
-  );
-  t.doesNotThrow(
-    createLogger.bind(null, {logLevel: '3'}),
-    createLogger({logLevel: '3'}),
-    ['Does not throw given a valid configuration object with a valid log level',
-    'in string form'].join(' ')
-  );
-  t.throws(
-    createLogger.bind(null, {logLevel: null}),
-    undefined,
-    ['Throws given a configuration object with the logLevel key specified that',
-    'is not of the correct type'].join(' ')
-  );
-  process.env.GCLOUD_ERRORS_LOGLEVEL = 4;
-  t.doesNotThrow(
-    createLogger,
-    createLogger({logLevel: 4}),
-    ['Does not throw given only the environment configuration variable for log',
-    'level setting'].join(' ')
-  );
-  process.env.GCLOUD_ERRORS_LOGLEVEL = oldEnv;
-  t.end();
+describe('logger', function () {
+  describe('Initialization', function () {
+    var oldEnv;
+    before(function () {
+      oldEnv = process.env.GCLOUD_ERRORS_LOGLEVEL;
+      delete process.env.GCLOUD_ERRORS_LOGLEVEL;
+    });
+    after(function () {process.env.GCLOUD_ERRORS_LOGLEVEL = oldEnv;});
+    describe('Exception handling', function () {
```

Comment:
Before/After happens after all the it's have been executed in the current and child describes